packages/apollo-server/src/index.ts

```diff
@@ -1,7 +1,6 @@
-import { NodeClient } from "@appsignal/types"
```

Comment:
I think we should keep the types. They're also useful for our users who use TypeScript and use types from our library. This way they don't need to specify `any` as argument types.

We can update the interfaces in types package in the javascript repo if needed.